core/src/test/java/io/questdb/griffin/O3PartitionPurgeTest.java

```diff
@@ -356,6 +356,71 @@ public void testPartitionDeletedAsyncAfterDroppedBySql() throws Exception {
         });
     }
 
+    @Test
```

Comment:
file doesn't seem to be formatted correctly

Reply:
fixed in https://github.com/questdb/questdb/pull/2802/commits/e8545c9e00a159392d3674a6c27f8840075fb742